packages/redux/src/raffles/selectors.ts

```diff
@@ -0,0 +1,224 @@
+  getEntityById(state, 'raffles', raffleId);
+
+/**
+ * Returns the estimation for a raffle
+ * @param state - Application state
+ * @param raffleId - Raffle id.
+ *
+ * @returns the object estimation raffle if exists.
+ */
+export const getRaffleEstimation = (
+  state: StoreState,
+  raffleId: Raffle['id'],
+) => getEntityById(state, 'raffleEstimations', raffleId);
+
+/**
+ * Return a raffle participation.
+ *
+ * @param state - Application state
+ * @param participationId - Participation id.
+ */
+export const getRaffleParticipation = (
+  state: StoreState,
+  participationId: RaffleParticipation['id'],
+) => getEntityById(state, 'raffleParticipations', participationId);
+
+/**
+ * Get a raffle participation for a specific raffle Id.
+ *
+ * @param state - Application state
+ * @param raffleId - raffle id.
+ *
+ * @returns a raffle participation if found otherwise undefined.
+ */
+export const getRaffleParticipationForRaffle = createSelector(
+  [
+    getRaffleParticipations,
+    (state: StoreState, raffleId: Raffle['id']) => raffleId,
+  ],
+  (raffleParticipations, raffleId) =>
+    raffleParticipations.find(
+      participation => participation.raffleId === raffleId,
+    ),
+);
+
+/**
+ * Return a raffle with estimation and participation details.
+ *
+ * @param state - Application state.
+ * @param raffleId - Raffle id.
+ *
+ */
+export const getDetailedRaffle = createSelector(
+  [getRaffle, getRaffleEstimation, getRaffleParticipationForRaffle],
+  (raffle, estimation, participation) => {
+    if (!raffle) {
+      return;
+    }
+    return {
+      ...raffle,
+      estimation,
+      participation,
+    };
+  },
+);
+
+/**
+ * Return all raffles loading from state.
+ *
+ * @param state - Application state.
+ *
+ */
+export const getIsLoadingAllRaffles = (state: StoreState) =>
+  getAllRaffles(state.raffles as RafflesState).isLoading;
+
+/**
+ * Return error raffles status from state.
+ *
+ * @param state - Application state.
+ *
+ */
+export const getErrorAllRaffles = (state: StoreState) =>
+  getAllRaffles(state.raffles as RafflesState).error;
+
+/**
+ * Return raffles loading from state.
+ *
+ * @param state - Application state.
+ * @param raffleId - Raffle Id to check.
+ *
+ * @returns isLoading state otherwise false.
+ */
+export const getIsLoadingRaffle = (state: StoreState, raffleId: Raffle['id']) =>
+  getRafflesFromReducer(state.raffles as RafflesState)[raffleId]?.isLoading ||
+  false;
+
+/**
+ * Return error raffles status from state.
+ *
+ * @param state - Application state.
+ * @param raffleId - Raffle Id to check.
+ *
+ */
+export const getErrorRaffle = (state: StoreState, raffleId: Raffle['id']) =>
+  getRafflesFromReducer(state.raffles as RafflesState)[raffleId]?.error;
+
+/**
+ * Return raffle estimation loading from state.
+ *
+ * @param state - Application state.
+ * @param raffleId - Raffle Id to check.
+ *
+ * @returns isLoading state otherwise false.
+ */
+export const getIsLoadingRaffleEstimation = (
```

Comment:
```suggestion
export const isRaffleEstimationLoading = (
```
Please, review the remaining selectors in this file to follow the same rule from the examples as I suggested here.

Reply:
done